pkgs/development/tools/parsing/tree-sitter/update_impl.py

```diff
@@ -213,7 +230,7 @@ def fetchAndCheckTreeSitterRepos() -> None:
 
 match mode:
     case "fetch-repo":
-        fetchRepo()
+        fetchRepo(MIN_TIMESTAMP)
```

Comment:
No big impact, but it could be better to pass the already parsed timestamp, instead of passing the string and recomputing it each time